### PR TITLE
update theme

### PR DESCRIPTION
**Pull request by OriolAbril**
3rd theme update PR, see https://github.com/pymc-devs/pymc/pull/6296 for more details

### CODE REVIEW

**Comment by drbenvincent** [comment]
Hi @OriolAbril. So I've been learning a bit more about sphinx, but there's so much to it that I'm obviously not a master yet! Do you just need an approval for this? The built docs look fine with no problems.

File: examples/conf.py [hunk 1]
```diff
@@ -112,35 +112,9 @@ def setup(app: Sphinx):
 # a list of builtin themes.
 
 # theme options
-html_theme = "pydata_sphinx_theme"
+html_theme = "pymc_sphinx_theme"
 html_theme_options = {
-    "icon_links": [
```
Comment (twiecki) [inline examples/conf.py:117]: Why are you removing these?

Comment (OriolAbril) [reply examples/conf.py:117]: answered in https://github.com/pymc-devs/pymc/pull/6296/files#r1022185629

**Comment by OriolAbril** [comment]
I would like some help double checking everything looks _and_ works the same, that all links are working, things like that. After that leave a review with comments or aproval

**Comment by twiecki** [comment]
Where can we view it?

**Comment by OriolAbril** [comment]
> Where can we view it?

Yes, alll 3 repos have a readthedocs "check", if you click on the "details" there you'll see the preview. For this PR the "details" link goes to https://pymcio--465.org.readthedocs.build/projects/examples/en/465/gallery.html

**Comment by twiecki** [comment]
image
the outside link box is back

**Comment by OriolAbril** [comment]
Everything should be fixed now

**Pull request merged**